crates/compiler/gen_dev/src/generic64/aarch64.rs

```diff
@@ -3069,7 +3212,7 @@ fn ldr_freg64_reg64_imm12(
 }
 
 #[inline(always)]
-fn ldr_freg64_reg64_imm9(
```

Comment:
Just a tad surprised by this, is our immediate always unsigned? I assume ldr is load signed and ldur is load unsigned.